diag_manager/README.md

```diff
@@ -93,10 +93,9 @@ In the legacy ascii format:
 
 **NOTE:** The fourth column (file_format) has been deprecated. Netcdf files will always be written.
 
-Below are some *optional* keys that may be added. 
+Below are some *optional* keys that may be added.
 - **write_file** is a logical that indicates if you want the file to be created (default is true). This is a new feature that is not supported by the legacy ascii data_table.
-- **new_file_freq** is a integer that defines the frequency for closing the existing file
-- **new_file_freq_units** is a string that defines the time units for creating a new file. Required if “new_file_freq” used. The acceptable values are seconds, minuts, hours, days, months, years. 
+- **new_file_freq** is a string that defines the frequency and the frequency units for closing the existing file
```

Comment:
**new_file_freq** is a string that defines the frequency and the frequency units (with a space between the frequency number and units) for closing the existing file